relay-server/src/services/processor.rs

```diff
@@ -1894,7 +1896,7 @@ impl EnvelopeProcessorService {
         }
 
         macro_rules! run {
-            ($fn:ident) => {{
+            ($fn_name:ident , $($args:expr),*) => {{
```

Comment:
to prevent trailing commas

```suggestion
            ($fn_name:ident $(, $args:expr)*) => {{
```